src/components/common/toast/toast.tsx

```diff
@@ -0,0 +1,73 @@
+import { Toast } from "@lifesg/react-design-system/toast";
+import { useEffect, useRef } from "react";
+import { EToastTypes, IToast, useDisplay } from "src/context-providers";
+import { ToastWrapper } from "./toast.styles";
+
+interface IProps {
+    toast?: IToast;
+    toastFunction?: () => void;
+    index?: number;
```

Comment:
how is index meant to be used? can remove if it has no use

Reply:
changes made in the last push